cli.js

```diff
@@ -41,5 +45,7 @@ if (process.stdin.isTTY) {
 
 	init(cli.input);
 } else {
-	process.stdin.pipe(isProgressive.stream(init));
+	isProgressive.stream(process.stdin).then(p => {
+		log(p, 'file');
+	});
```

Comment:
Good point! What do you want as message? Is this ok or rather prefer something like `input`, `stdin`, ...